lib/hyperswitch.js

```diff
@@ -362,8 +362,8 @@ RESTBase.prototype._request = function(req, options) {
     }
 
     if (handler) {
-        // Prepare to call the handler with a child restbase instance
-        var childRESTBase = this.makeChild(childReq, options);
+        // Prepare to call the handler with a child HyperSwitch instance
+        var childHyper = this.makeChild(childReq, options);
```

Comment:
euh, `childHyperSwitch` is a bit too long, and `childHyper` is more in line with the var name `hyper`, IMHO.